Makefile

```diff
@@ -0,0 +1,37 @@
+ENVIRONMENT ?= betacloud
+
+VERSION_CEPH ?= pacific
+VERSION_MANAGER ?= latest
+VERSION_OPENSTACK ?= yoga
+
+clean:
+	cd terraform && make ENVIRONMENT=$(ENVIRONMENT) clean
+
+create:
+	cd terraform && make ENVIRONMENT=$(ENVIRONMENT) VERSION_CEPH=$(VERSION_CEPH) VERSION_MANAGER=$(VERSION_MANAGER) VERSION_OPENSTACK=$(VERSION_OPENSTACK) create
+
+login:
+	cd terraform && make ENVIRONMENT=$(ENVIRONMENT) login
+
+deploy:
+	ansible-playbook playbooks/deploy.yml \
+	  -i ansible/localhost_inventory.yaml \
+	  -e ansible_galaxy=ansible-galaxy \
+	  -e ansible_playbook=ansible-playbook \
+	  -e basepath="$(PWD)" \
+	  -e cloud_env=$(ENVIRONMENT) \
+	  -e repo_path="$(PWD)/.src/github.com"
+	  -e manual_deploy=true \
+	  -e version_ceph=$(VERSION_CEPH) \
+	  -e version_manager=$(VERSION_MANAGER) \
+	  -e version_openstack=$(VERSION_OPENSTACK)
+	cd terraform && make ENVIRONMENT=$(ENVIRONMENT) deploy
+	cd terraform && make ENVIRONMENT=$(ENVIRONMENT) check
+
+prepare:
+	mkdir -p .src/github.com/osism
+	if [[ ! -e .src/github.com/osism/testbed ]]; then git clone https://github.com/osism/testbed .src/github.com/osism/testbed; fi
```

Comment:
Since this Makefile is already within the testbed repo, creating yet another clone of the repo seems wasteful. Can we just use the copy we are running in instead?

Reply:
I did this on purpose so that the credentials and state would not be copied from this working copy.